custom-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,421 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tikit",
+    "description": "This is a ticketing management system.",
```

Comment:
@1MrSharma Please provide more details about your service or product which can be understandable by everyone and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
Updated description for the connector that is more understandable.